README.md

```diff
@@ -9,15 +9,11 @@
 ## MFTF Setup
 1. Clone pwa-tests repository
 2. Run `composer install`
+3. Copy .env file and update the variables. To copy `cp dev/tests/acceptance/.env.dist dev/tests/acceptance/.env`
 3. Run `vendor/bin/mftf build:project`
 
 ## Running Tests
-1. Run Sample Tests:
+1. Running Tests Ex:
```

Comment:
Can probably just get rid of the num list, or fix this name (assume you mean `Examples` and not `Ex`.

Reply:
Done